packages/ui/src/components/form-primitives/form-caption.tsx

```diff
@@ -0,0 +1,47 @@
+import { PropsWithChildren } from 'react'
+
+import { Icon } from '@components/icon'
+import { cn } from '@utils/cn'
+import { cva, VariantProps } from 'class-variance-authority'
+
+const formCaptionVariants = cva('cn-caption', {
+  variants: {
+    theme: {
+      default: '',
+      success: 'cn-caption-success',
+      danger: 'cn-caption-danger',
+      warning: 'cn-caption-warning'
+    }
+  },
+  defaultVariants: {
+    theme: 'default'
+  }
+})
+
+type FormCaptionProps = {
+  theme?: VariantProps<typeof formCaptionVariants>['theme']
+  className?: string
+}
+
+export const FormCaption = ({ theme = 'default', className, children }: PropsWithChildren<FormCaptionProps>) => {
+  if (!children) {
+    return null
+  }
+
+  const canShowIcon = theme === 'danger' || theme === 'warning'
+
+  /**
+   * cross-circle - danger
+   * triangle-warning - warning
+   */
+  const effectiveIconName = theme === 'danger' ? 'cross-circle' : 'warning-triangle-outline'
+
+  return (
+    <p role="region" aria-live="polite" className={cn(formCaptionVariants({ theme }), className)}>
```

Comment:
`aria-live="polite"` is meant for announcing changes to out-of-focus elements for vision-impaired users. Why is it added here?
Similar question for `region`

Reply:
If some error occurs, then it will announce that will help vision-impaired users. In that thought I added. I think we can remove it for now and add accessibility related attributes selectively.